### DISCUSSION TITLE
Allow Logbeat connections for 2.4 firewall

### DISCUSSION THREAD

**Discussion by kgoode517** [Security-Onion-Solutions/securityonion#11114] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am attempting to hook up beats to my standalone's logstash for a few of my older clients that have to use filebeat and metericbeat instead of elastic agent  but can't seem to get 5044 to accept incoming connections on the new 2.4 onion. I followed the port group firewall documnetation to open both the beats port and tcp port 5044 as a custom port but this does not seem to work to allow a connection. I've checked the iptables rules and they look good to me.   Suggestions and or insights would be much appericated.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment, marked as answer]
This has not been tested on 2.4. We are not sure that logs will be parsed correctly, and it is likely that you will run into version issues.
